docs/source/user/api_change.rst

```diff
@@ -9,7 +9,105 @@ The changes are tabulated according to the module input file, line number, and f
+input file.  See examples in the regression tests for the new formats.
+
+New modules AeroDisk (see :numref:`ADsk`) and Simplified-ElastoDyn (see :numref:`SED`)
+were added.  See documentation on those modules for exmple input files.
+
+============================================= ======= ==================== ========================================================================================================================================================================================================
+Modified in OpenFAST `dev`
+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+Module                                        Line    Flag Name            Example Value
+============================================= ======= ==================== ========================================================================================================================================================================================================
+OpenFAST                                      15      CompAero\**          2   CompAero        - Compute aerodynamic loads (switch) {0=None; 1=AeroDisk; 2=AeroDyn; 3=ExtLoads}
+OpenFAST                                      13      CompElast            3   CompElast       - Compute structural dynamics (switch) {1=ElastoDyn; 2=ElastoDyn + BeamDyn for blades; 3=Simplified ElastoDyn}
+AeroDyn                                       40      IntegrationMethod    3   IntegrationMethod  - Switch to indicate which integration method UA uses (1=RK4, 2=AB4, 3=ABM4, 4=BDF2)
+AeroDyn                                       80\*     NacArea             0, 0, 0            NacArea            - Projected area of the nacelle in X, Y, Z in the nacelle coordinate system (m^2)
+AeroDyn                                       81\*     NacCd               0, 0, 0            NacCd              - Drag coefficient for the nacelle areas defined above (-)
+AeroDyn                                       82\*     NacDragAC           0, 0, 0            NacDragAC          - Position of aerodynamic center of nacelle drag in nacelle coordinates (m)
+AeroDyn                                       140\*   BldNd_BlOutNd        "All"  BldNd_BlOutNd   - Specify a portion of the nodes to output. {"ALL", "Tip", "Root", or a list of node numbers} (-)
+AeroDyn Aeroacoustics                         11\*    TI                   0.1 TI   - Rotor-incident wind turbulence intensity (-) [Only used if TiCalcMeth == 1]
+AeroDyn Aeroacoustics                         12\*    avgV                 8 avgV   - Average wind speed used to compute the section-incident turbulence intensity (m/s) [Only used if TiCalcMeth == 1]
+ElastoDyn blade file                          15                           Removal of the `PitchAxis` input column
```

Comment:
A new section was added to the ElastoDyn primary input file to support yaw friction.  Also, cross-inertia terms have been added for the platform.

Reply:
Updated in [e973b76](https://github.com/OpenFAST/openfast/pull/2586/commits/e973b76cf68077910f974fb03a2c963c6f49cbad)